lighthouse-core/audits/manifest-background-color.js

```diff
@@ -36,22 +37,27 @@ class ManifestBackgroundColor extends Audit {
    * @param {!Manifest=} manifest
    * @return {boolean}
    */
-  static hasBackgroundColorValue(manifest) {
+  static getBackgroundColorValue(manifest) {
     return manifest !== undefined &&
       manifest.background_color !== undefined &&
-      manifest.background_color.value !== undefined;
+      manifest.background_color.value;
```

Comment:
apparently that was on an outdated diff :) from before: "for what it's worth, `manifest.background_color !== undefined` isn't necessary here"